packages/ripple-address-codec/src/xrp-codec.ts

```diff
@@ -161,7 +161,7 @@ const ED25519_SEED = [0x01, 0xe1, 0x4b]
 
 const codecOptions = {
   sha256(bytes: Uint8Array): Buffer {
-    return createHash('sha256').update(Buffer.from(bytes)).digest()
+    return Buffer.from(hashjs.sha256().update(bytes).digest())
```

Comment:
when I originally made this library, the hash function was configurable iirc 
"bring your own hash" :)

hashjs will be a bit of a perf drop for some use cases ?

Reply:
Strange, createHash should use crypto when running on node, then use the "browser" versions in the browser. 
Maybe you are using it in a strange environment and are missing some polyfills that some build tools like webpack usually automatically bundle.

Reply:
I think it's because it's compiled for Tauri ... you're right though I should try to get that working again. I had seen this as a quick route without downsides, but maybe that's not the case in practice.